### PR TITLE
added code of conduct for review

### PR DESCRIPTION
**Pull request by kiihne-noaa**
## Describe your changes
Created a .md file for a code of conduct that is current under construction.

### CODE REVIEW

**Review by ceblanton — APPROVED** [review]
Nice! I'm not sure about gfdl.climate.model.info@noaa.gov as a HR contact, though. That address is used for questions or problems with externally-visible climate model output, by convention.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Addressing Grievances:
+Unacceptable behavior from any community member will not be tolerated. Anyone asked to stop unacceptable behavior is expected to comply immediately. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by emailing gfdl.climate.model.info@noaa.gov 
+
+## Attribution
+This Code of Conduct is adapted from the [Contributor Covenant], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html, the [Citizen Code of Conduct] available at https://web.archive.org/web/20200330154000/http://citizencodeofconduct.org/, and the [GFDL Code of Conduct] available at https://www.gfdl.noaa.gov/wp-content/uploads/2022/05/GFDL_Code_of_Conduct.pdf
```
Comment (ilaflott) [inline CODE_OF_CONDUCT.md:30]: are `[Contributor Covenant]`, `[Citizen Code of Conduct]`, and `[GFDL Code of Conduct]` supposed to be hyperlinks that point you to the links?

To make it clickable then, one would do, e.g. `[Contributor Covenant](Contributor Covenant url)`, like this: [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html)

Comment (ceblanton) [reply CODE_OF_CONDUCT.md]: The links all work now, but it's odd that the Citizen Code of Conduct is only available on the Internet Archive. Is the Citizen Code of Conduct available somewhere else?

**Comment by ceblanton** [comment]
@kiihne-noaa are you happy with this or are there other updates coming?

**Comment by kiihne-noaa** [comment]
> @kiihne-noaa are you happy with this or are there other updates coming?

I am satisfied. This can be closed.

**Pull request merged**